sdk/textanalytics/Azure.AI.TextAnalytics/tests/samples/Sample7_AnalyzeHealthcareEntitiesBatchConvenience.cs

```diff
@@ -0,0 +1,115 @@
+                new TextDocumentInput("3", string.Empty)
+            };
+            var options = new AnalyzeHealthcareEntitiesOptions { IncludeStatistics = true };
+
+            AnalyzeHealthcareEntitiesOperation healthOperation = client.StartAnalyzeHealthcareEntities(batchInput, options);
+
+            await healthOperation.WaitForCompletionAsync();
+
+            Console.WriteLine($"AnalyzeHealthcareEntities operation was completed");
+
+            Console.WriteLine($"Created On   : {healthOperation.CreatedOn}");
+            Console.WriteLine($"Expires On   : {healthOperation.ExpiresOn}");
+            Console.WriteLine($"Id           : {healthOperation.Id}");
+            Console.WriteLine($"Status       : {healthOperation.Status}");
+            Console.WriteLine($"Last Modified: {healthOperation.LastModified}");
+
+            foreach (AnalyzeHealthcareEntitiesResultCollection documentsInPage in healthOperation.GetValues())
+            {
+                Console.WriteLine($"Results of Azure Text Analytics \"Healthcare\" Model, version: \"{documentsInPage.ModelVersion}\"");
+                Console.WriteLine("");
+
+                int i = 0;
+
+                foreach (AnalyzeHealthcareEntitiesResult result in documentsInPage)
+                {
+                    TextDocumentInput document = batchInput[i++];
+
+                    Console.WriteLine($"On document (Id={document.Id}, Language=\"{document.Language}\"):");
+
+                    if (result.HasError)
+                    {
+                        Console.WriteLine("  Error!");
+                        Console.WriteLine($"  Document error code: {result.Error.ErrorCode}.");
+                        Console.WriteLine($"  Message: {result.Error.Message}");
+                    }
+                    else
+                    {
+                        Console.WriteLine($"    Recognized the following {result.Entities.Count} healthcare entities:");
+
+                        foreach (HealthcareEntity entity in result.Entities)
+                        {
+                            Console.WriteLine($"    Entity: {entity.Text}");
+                            Console.WriteLine($"    Category: {entity.Category}");
+                            Console.WriteLine($"    Offset: {entity.Offset}");
+                            Console.WriteLine($"    Length: {entity.Length}");
+                            Console.WriteLine($"    Links:");
+
+                            foreach (EntityDataSource entityDataSource in entity.DataSources)
+                            {
+                                Console.WriteLine($"        Entity ID in Data Source: {entityDataSource.EntityId}");
+                                Console.WriteLine($"        DataSource: {entityDataSource.Name}");
+                            }
+                        }
+                        Console.WriteLine("");
+                    }
+
+                    Console.WriteLine($"Batch operation statistics:");
```

Comment:
because the idea is that in the convenience we don't show statistics, all the printing for statistics should go away